### PR TITLE
Adding ** to call.

### PR DESCRIPTION
**Pull request by adrianmann**
Please review this code to address the following issue:

https://github.com/kreeti/kt-paperclip/issues/46

I ran my test suite against these changes and I no longer see the warnings. I also ran the kt-paperclip tests an they are passing.

### CODE REVIEW

**Comment by adrianmann** [comment]
The last build failed but is not failing locally for me. Could this be a random failure? I can't restart the build if so.

File: lib/paperclip/validators/attachment_file_name_validator.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def validate_blacklist(record, attribute, value)
       end
 
       def mark_invalid(record, attribute, patterns)
-        record.errors.add attribute, :invalid, options.merge(names: patterns.join(", "))
+        record.errors.add attribute, :invalid, **options.merge(names: patterns.join(", "))
```
[the remark this answers is not in the document]
Comment (adrianmann) [reply lib/paperclip/validators/attachment_file_name_validator.rb:46]: Do I need to address the feedback from the bot?

[the remark this answers is not in the document]
Comment (ssinghi) [reply lib/paperclip/validators/attachment_file_name_validator.rb:46]: no. the bot config needs to be fixed.

**Comment by ssinghi** [comment]
@adrianmann Thanks for your PR. Can you plz remove the NEWS commit? The file isn't being maintained. This will trigger another build as well, and we will know if it was just a random fragile test failure. Thanks!

**Comment by adrianmann** [comment]
I have removed the change to the NEWS file in the latest commit and tests passed this time, please review again.

**Comment by adrianmann** [comment]
Bump! :)

**Comment by adrianmann** [comment]
Thanks for merging this in, it seems the last build on master has failed. Can the build be restarted, I'd imagine it is an intermittent failure?

**Comment by Kevinrob** [comment]
@ssinghi Is it possible to have a new release with this code please?

**Comment by rstueven** [comment]
@adrianmann Thanks!

Apparently my project uses the defunct **paperclip-av-transcoder** gem, and it relies on **kt-paperclip** ~> 6.4, so I'll have to figure out a workaround.

**Pull request merged**